.release-notes/4147.md

```diff
@@ -0,0 +1,14 @@
+## Change error message
+
+error message in the situation:
```

Comment:
I would try to write a summary in english before any code blocks, e.g.

The error message when assigning to a field of a field in `box` functions was fixed to reflect that the function should be a `fun ref`. This affects code like the below:

Reply:
ill add it in, thanks!